oauth2_http/java/com/google/auth/oauth2/storage/TokenStore.java

```diff
@@ -29,7 +29,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package com.google.auth.oauth2;
+package com.google.auth.oauth2.storage;
```

Comment:
Could we leave this in the original package? I'd prefer not to make a breaking change for this.